timeline-chart/src/time-graph-font-controller.ts

```diff
@@ -6,8 +6,8 @@ export class FontController {
     private fontColorMap: Map<number, string>;
     private defaultFontName: string = "LabelFont8White";
 
-    constructor(fontFamily: string = "\"Lucida Console\", Monaco, monospace") {
-        this.fontFamily = fontFamily;
+    constructor() {
```

Comment:
Why not keep it in the constructor as currently, so it's configurable by callers?

Reply:
That is better. Thank you.

Reply:
Modified the constructor code.